CMakeLists.txt

```diff
@@ -135,3 +135,17 @@ target_link_libraries(tests ${BLAS_LIBRARIES})
 target_link_libraries(tests ${OpenCL_LIBRARIES})
 target_link_libraries(tests ${ZLIB_LIBRARIES})
 target_link_libraries(tests gtest_main ${CMAKE_THREAD_LIBS_INIT})
+
+SET(MAJOR_VERSION 0)
+SET(MINOR_VERSION 15)
```

Comment:
It is not acceptable to hardcode this here.

Reply:
Have a look at this:
https://github.com/rpavlik/cmake-modules/blob/master/GetGitRevisionDescription.cmake

Reply:
Here is also a good read.

http://ipenguin.ws/2012/11/cmake-automatically-use-git-tags-as.html

Including GetGitRevisionDescription is a big change to do because you have to add the module under this project. Making debian package should be independent to the work of automatically versioning.

I am not sure where to put the GetGitRevisionDescription module in this project, so I will stop here unless we have a good way to get leela-zero version in cmake.

Reply:
I'm not sure how hard it is to get it from the config.h header?

Reply:
We need a single source of version number. Maybe the version number can be defined in a configure file though, there are more than one way to “make” in this project. Thinking what the best way is...

Reply:
This project does not have ./configure to define a global version number, so I prefer to use GetGitRevisionDescription.cmake to get git tags as version strings. To be done...

Reply:
I referred to the config.h header as that is where the program itself gets its version number from, so that would be a stable source. If you are willing to work with GetGitRevisionDescription, then that's even better (it's also a stable source).